src/internal_events/loki.rs

```diff
@@ -23,7 +23,7 @@ impl InternalEvent for LokiEventsProcessed {
 }
 
 #[derive(Debug)]
-pub struct LokiUniqueStream;
+struct LokiUniqueStream;
```

Comment:
This looks odd. What is the point of unexporting this?

Reply:
I think this is actually unused but it doesn't trigger an unused warning. The only use site is in this file, its definition and the impl for `InternalEvent`.

Reply:
I've pushed https://github.com/vectordotdev/vector/pull/11458/commits/5b759247151e5044e42216b9a6e971e1c21764c7 to see if this breaks the build, removing the struct entirely. 

Very confounded as to why this doesn't raise an unused warning.